packages/commonwealth/server/migrations/20230616214402-remove-duplicate-eth-chain-node.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.bulkUpdate(
+        'Chains',
+        {
+          chain_node_id: 37,
+        },
+        {
+          chain_node_id: 1285,
```

Comment:
What's the reason for this odd chain_node_id?

Reply:
This updates all the chains/communities that point to the incorrect/false Ethereum chain node (1285) to point to the correct (37) chain node. Not sure if that answers this question.